qa/vagrant/src/test/resources/packaging/scripts/70_sysv_initd.bats

```diff
@@ -117,6 +117,23 @@ setup() {
     [ "$status" -eq 3 ] || [ "$status" -eq 4 ]
 }
 
+@test "[INIT.D] don't mkdir when it contains a comma" {
+    # Remove these just in case they exist beforehand
+    rm -rf /tmp/aoeu,/tmp/asdf
+    rm -rf /tmp/aoeu,
+    # set DATA_DIR to DATA_DIR=/tmp/aoeu,/tmp/asdf
+    sed -i 's/DATA_DIR=.*/DATA_DIR=\/tmp\/aoeu,\/tmp\/asdf/' /etc/init.d/elasticsearch
+    cat /etc/init.d/elasticsearch | grep "DATA_DIR"
+    service elasticsearch start
+    wait_for_elasticsearch_status
+    assert_file_not_exist /tmp/aoeu,/tmp/asdf
```

Comment:
I can't tell if you're agreeing with me or not? I think that the `tree` output is misleading here. The second level `tmp` is a subdirectory of `/tmp/aoeu,/` It's clearer if you do `tree /tmp`:

```bash
ubuntu-1404:~$ tree /tmp
/tmp
└── aoeu,
    └── tmp
        └── asdf

3 directories, 0 files
ubuntu-1404:~$ tree /tmp/aoeu,/
/tmp/aoeu,/
└── tmp
    └── asdf
2 directories, 0 files
```

Note in the system calls, first it tries to `mkdir` on `/tmp`, and then it `chdir` on `/tmp`, then it `mkdir` on `aoeu,` and then `open` on `aoeu,` which gives make a file descriptor `3` which it `fchdir` into, and them `mkdir tmp` there (so now we are in `/tmp/aoeu,`) and then finally `open` on `tmp` so it can `fchdir` again where it `mkdir` on `asdf`.